internal/api/impl/v1/globalrole/service.go

```diff
@@ -78,11 +84,17 @@ func (s *service) update(entity *v1.GlobalRole, parameters apiInterface.Paramete
 		logrus.WithError(updateErr).Errorf("unable to perform the update of the Globalrole %q, something wrong with the database", entity.Metadata.Name)
 		return nil, updateErr
 	}
+	if err := s.rbac.Refresh(); err != nil {
+		return nil, err
+	}
 	return entity, nil
 }
 
 func (s *service) Delete(_ apiInterface.PersesContext, parameters apiInterface.Parameters) error {
-	return s.dao.Delete(parameters.Name)
+	if err := s.dao.Delete(parameters.Name); err != nil {
+		return err
+	}
+	return s.rbac.Refresh()
```

Comment:
it would be weird to return an error because the refresh cache failed right ? Specially because the resources has been successfully deleted

Reply:
actually everytime we are refreshing the cache and it fails probably we should just log the error and not returning it to the user